src/Evolution/DiscontinuousGalerkin/Limiters/Weno.hpp

```diff
@@ -58,42 +58,106 @@ namespace Limiters {
+/// troubled, then all components of all tensors are limited.
+///
+/// When the evolution system has multiple evolved variables, the recommendation
+/// of the references is to apply the limiter to the system's characteristic
+/// variables to reduce spurious post-limiting oscillations. In SpECTRE,
+/// applying the limiter to the characteristic variables requires specializing
+/// the limiter to each evolution system. The system-specific limiter can also
+/// implement a system-specific TCI (as the HWENO reference does) to more
+/// precisely trigger the limiter.
+///
+/// ##### 3. The oscillation indicator
+///
+/// We use the oscillation indicator of \cite Dumbser2007, modified for use on
+/// the square/cube grids of SpECTRE. We favor this indicator because portions
+/// of the work can be precomputed, leading to an oscillation measure that is
 /// efficient to evaluate.
 ///
-/// \warning
-/// Limitations:
-/// - Does not support non-Legendre bases; this is ASSERTed.
-/// - Does not support h- or p-refinement; this is ASSERTed.
-/// - Does not support curved elements; this is not enforced.
+/// ##### 4. The geometric limitations
+///
+/// Does not support non-Legendre bases; this is checked in DEBUG mode. In
+/// principle other bases could be supported, but this would require
+/// generalizing the many internal algorithms that assume a Legendre basis.
+///
+/// Does not support h- or p-refinement; this is checked always. In principle
+/// this could be supported. The modified neighbor solution algorithm would
+/// need to be generalized (reasonable for simple WENO, extremely tedious for
+/// HWENO), and the sum of neighbor solutions may need to be updated as well.
+///
+/// Does not support curved elements; this is not enforced. The code will run
+/// but we make no guarantees about the results. Specifically, the limiter acts
+/// in the `Frame::Logical` coordinates, because in these coordinates it is
+/// straightforward to formulate the algorithm. This means the limiter can
+/// operate on generic deformed grids --- however, some things can start to
+/// break down, especially on strongly deformed grids:
+/// 1. When the Jacobian (from `Frame::Logical` to `Frame::Inertial`) varies
+///    across the element, then the limiter fails to be conservative. This is
+///    because the integral of a tensor `u` over the element will change after
+///    the limiter activates on `u`.
+/// 2. When computing the modified neighbor solution for the WENO sum, the
+///    extrapolation or fitting procedure may not properly account for the
+///    coordinates of the source data. If the coordinate map of the neighbor
+///    differs from that of the local element, then the logical-coordinate
+///    representation of the neighbor data may be incorrect. This may be a
+///    large error at Block boundaries with discontinuous map changes, and may
+///    be a small error from smoothly-varrying maps that are not sufficiently
```

Comment:
ah man, typo in the last sentence "varying"

Reply:
ah man...